docs/ko/week03/03-1.md

```diff
@@ -0,0 +1,371 @@
+-->
+실제로는 PyTorch에서의 합성곱은 **반전되지 않은** $w$로 구현되어 조금 다른 관례<sup>Convention</sup>를 따랐다.
+
+$$y_i = \sum_j w_j x_{i+j}$$
+<!--
+Mathematicians call this formulation "cross-correlation". In our context, this difference is just a difference in convention. Practically, cross-correlation and convolution can be interchangeable if one reads the weights stored in memory forward or backward.
+-->
+수학자들은 이 공식을 "교차 상관관계"라고 부른다. 합성곱과의 차이점은 단지 "관례"의 차이만 있을 뿐이다. 실제로 메모리에 저장된 가중치를 앞으로 읽는지 뒤로 읽는지에 따라 교차상관과 합성곱을 서로 바꿔 쓸 수 있다.
+
+<!--
+Being aware of this difference is important, for example, when one want to make use of certain mathematical properties of convolution/correlation from mathematical texts.
+-->
+예를 들어, 합성곱과 교차상관의 수학적 성질을 사용하고자 할 때 이러한 차이점을 인식하는 것이 중요하다.
+
+
+### 고차원 합성곱 (Higher dimensional convolution)
+---
+<!--
+For two dimensional inputs such as images, we make use of the two dimensional version of convolution:
+-->
+이미지와 같이 2차원 입력의 경우에는 2차원 버전의 합성곱을 사용한다.
+
+$$y_{ij} = \sum_{kl} w_{kl} x_{i+k, j+l}$$
+
+<!--
+This definition can easily be extended beyond two dimensions to three or four dimensions. Here $w$ is called the *convolution kernel*
+-->
+위 정의는 2차원을 넘어 3차원 또는 4차원으로 쉽게 확장할 수 있다. 여기서 $w$는 *합성곱 커널*<sup>Convolution Kernel</sup>이다.
+
+
+### DCNN에서 합성곱 연산과 함께 쓰는 Regular twists
+---
+
+| 스트라이드 | 1 | 2 |
+|: ------------ :|: -------------------------- :|: -------------------------- :|
+| 출력 크기: | $\frac{100 - (5-1)}{1}=96$ | $\frac{100 - (5-1)}{2}=48$ |
+
+<!--
+1. **Striding**: instead of shifting the window in $x$ one entry at a time, one can do so with a larger step (for example two or three entries at a time). Example: Suppose the input $x$ is one dimensional and has size of 100 and $w$ has size 5. The output size with a stride of 1 or 2 is shown in the table below:
+2. **Padding**: Very often in designing Deep Neural Networks architectures, we want the output of convolution to be of the same size as the input. This can be achieved by padding the input ends with a number of (typically) zero entries, usually on both sides. Padding is done mostly for convenience. It can sometimes impact performance and result in strange border effects, that said, when using a ReLU non-linearity, zero padding is not unreasonable.
+-->
+
+1. **스트라이드**<sup>Striding</sup>: $x$에서 윈도우를 한 번만 이동시키는 것 대신에, 한 번에 두 세번씩 더 큰 스텝을 진행할 수 있다.
+예시: 입력 $x$가 1차원이고 크기가 100이며, $w$의 크기가 5라고 가정하자. 위 표는 스텝이 1 또는 2인 출력 크기를 보여주고 있다.
+
+2. **패딩**<sup></sup>: 보통 심층 신경망<sup>Deep Neural Networks</sup> 아키텍처를 설계할 때, 입력의 양 끝단에 0으로 채워주는(일반적으로 0, 0이 아닐 수도 있음) 패딩을 해줌으로써 합성곱의 출력을 입력과 같은 크기로 만들어준다.
+
+```
+It can sometimes impact performance and result in strange border effects, that said, when using a ReLU non-linearity, zero padding is not unreasonable.
+주로 패딩은 우리의 편의를 위해 사용하는 것이며, 때로는 성능에 영향을 주고, 이상한 Border Effect를 가져올 수 있다. 즉, ReLU 비선형성을 사용할 때에는 제로 패딩<sup>Zero padding</sup>을 사용하는 것은 별로 좋지 않다.
+```  
```

Comment:
`패딩은 우리의 편의를 위해 사용하는 것이며` 원문에 존재하지 않고, 편의상 사용하는 것도 아닙니다.

또한 `zero padding is not unreasonable` 0패딩은 unreasonable하지 않다고 하고 있습니다.

보통 미디어 처리의 합성곱에서 0 패딩은 이미지 바깥쪽에 나쁜 border effect를 줘서 잘 사용하지 않지만, CNN에서 relu non-linear 함수와는 주로 사용합니다!

Reply:
하하 원문에 `It can some times` 전에 ` Padding is done mostly for convenience.`라는 부분이 존재하여  번역을 하였고 출력 크기글 맞춰주려고 쓰는 거니깐 편의를 위해 사용하는 것도 맥락엔 맞지 않을까 싶어요.

`not unreasonable`인데 `un`을 미처 보지 못하였군요.. 수정 반영해야겠네요.